kod/object/item/passitem/ring/lethring.kod

```diff
@@ -118,7 +118,6 @@ messages:
       % at this point, it's pretty obvious it's not helping them.
       if Send(self, @RevealHiddenAttributes)
       {
-         Send(self, @RevealHiddenColor);
```

Comment:
Note: This is removed because my PR calls RevealHiddenColor() from RevealHiddenAttributes().